config/environments/production.rb

```diff
@@ -27,8 +27,9 @@
   # Compress CSS using a preprocessor.
   # config.assets.css_compressor = :sass
 
-  # Do not fallback to assets pipeline if a precompiled asset is missed.
-  config.assets.compile = false
+  # We are pre-compiling assets, so on production these will always be present, however
+  # for this to work this flag needs to be `true`. Seems like a Rails bug to me.
```

Comment:
Will probably apply to review in the same way - thanks for flagging 👍

Reply:
Yep I believe you may have the same issue as we both are using the same assets precompile process and pipeline.